src/framework/lib/genesisframework/utils.rb

```diff
@@ -44,9 +44,12 @@ def self.facter
         end
       end
 
-      def self.log subsystem, message
-        logline = subsystem.to_s + " :: " + message
-        puts logline
+      def self.log subsystem, message, level = nil
```

Comment:
This should probably use things like https://github.com/tumblr/collins/blob/master/support/ruby/collins-client/lib/collins/api/logging.rb#L13-L21 to do the mapping from user provided severity to validated symbol, with methods like `#value_of` and `#level`